jena-fuseki2/jena-fuseki-ui/src/components/ServerStatus.vue

```diff
@@ -61,6 +61,7 @@ export default {
 
   async created () {
     this.serverStatus = await this.$fusekiService.getServerStatus()
+    console.log(this.serverStatus.message)
```

Comment:
Does this print to the console in normal operation?

Reply:
In the tests output. But if used in a component, then it prints to the browser console (the linter should fail if a console log is added to the main code, unless suppressed).